crates/solidity/testing/snapshots/cst_output/Statement/throw/generated/0.5.0-success.yml

```diff
@@ -0,0 +1,13 @@
+# This file is generated automatically by infrastructure scripts. Please don't edit by hand.
+
+Source: >
+  1  │ throw;                                                                           │ 0..6
+
+Errors: []
+
+Tree:
+  - Statement (Rule): # 0..6 "throw;"
+      - ExpressionStatement (Rule): # 0..6 "throw;"
+          - Expression (Rule): # 0..5 "throw"
+              - Identifier (Token): "throw" # 0..5
```

Comment:
I think this is broken. `throw` is now an `Identifier`?

Reply:
Yep, after a certain version. But this is also a `Deprecated` | `Reserved` issue, potentially,

Reply:
`int throw = 0;` is illegal in `0.8.21`. I just checked on https://remix.ethereum.org/

Reply:
`function throw() external pure  {}` would also produce `ParserError: Expected identifier but got 'throw'`

Reply:
Fixed